.prettierignore

```diff
@@ -0,0 +1,11 @@
+.DS_Store
+node_modules
+/build
+/.svelte-kit
+
+# Ignore files for PNPM, NPM and YARN
+pnpm-lock.yaml
+package-lock.json
+yarn.lock
+
+src/lib/data
```

Comment:
Could you add a newline at the end of this file?

Also, I think the following path should be added to the ignore file:
```
scripts/env
```
This is because the README recommends creating a Python environment in this directory for running the download scripts and we probably don't want Prettier to check those.